deeplabcut/pose_estimation_pytorch/apis/convert_detections_to_tracklets.py

```diff
@@ -42,9 +42,9 @@ def convert_detections2tracklets(
     ignore_bodyparts: Optional[List[str]] = None,
     inferencecfg: Optional[dict] = None,
     modelprefix="",
-    greedy=False,  # TODO: Unused, remove
-    calibrate=False,  # TODO: Unused, remove
-    window_size=0,  # TODO: Unused, remove
+    greedy: bool = False,  # TODO: Unused, remove
+    calibrate: bool = False,  # TODO: Unused, remove
+    window_size: int = 0,  # TODO: Unused, remove
```

Comment:
These arguments are planned to be added, right? Then maybe we should update the comments, to say that it isn't implemented yet, but will in the future

Reply:
Yes those will be added in a subsequent pull request.

```suggestion
    greedy: bool = False,  # TODO(niels): implement greedy assembly during video analysis
    calibrate: bool = False,  # TODO(niels): implement assembly calibration during video analysis
    window_size: int = 0,  # TODO(niels): implement window size selection for assembly during video analysis
```